.travis.yml

```diff
@@ -4,7 +4,6 @@ node_js:
 install:
   - npm install
 script:
-  - npm run package
```

Comment:
In this case, do you think we should switch this out with prepublish? In that, we probably want to run a sanity test that writing the config and running the build doesn't error out.